### DISCUSSION TITLE
UGS Launch Issue

### DISCUSSION THREAD

**Discussion by amarciniecCT** [winder/Universal-G-Code-Sender#1796] [Help] [answered]
Hello!

 I'm having trouble getting UGS to launch on my Windows 10 machine. I'm running Java 1.8.0_321, and each time I try to run UGS, I get the splash screen, followed by a message informing me that this is the Nightly Build. The program is unresponsive the entire time, then crashes after a second or two. Interestingly enough, I first had this issue with build 2.0.9. I've since tried build 2.0.8, as well as the actual nightly build. I experience the exact same issue each time. 

I've attached the error log from my most recent attempt, which was with build 2.0.8. I'd appreciate any help here! 

Thanks!
-Albert 
 
[hs_err_pid6708.log](https://github.com/winder/Universal-G-Code-Sender/files/7990312/hs_err_pid6708.log)

**Comment by breiler** [comment, marked as answer]
Hi Albert, can you attach the log located here:
C:/Users/[your username>]/.ugsplatform/2.X/var/log/messages.log

**Reply by amarciniecCT** [reply]
I certainly can. 

[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/7995144/messages.log)

**Reply by breiler** [reply]
That didn't give any clues...

Looking back at the hs_err_xxxx.log it looks like its crashing when it tries to query for graphic cards. Could be a driver issue or a problem with a antivirus software? I really don't know. =(

We could try to disable the OpenGL-support. Not sure how to do this in Windows, but I think you could run the application from the command line running these commands:
```
set LIBGL_ALWAYS_SOFTWARE=true
C:\Users\amarciniec\Desktop\ugsplatform-win\bin\ugsplatform.exe
```

**Reply by amarciniecCT** [reply]
Hmmm okay. Let me try updating graphics drivers, and I'll reach out to my company IT to see if it's maybe a firewall issue. If neither of those work, I'll give the OpenGL thing a shot. I'll update if anything changes, and I appreciate your help all the same!

**Reply by amarciniecCT** [reply]
Ok, so I tried to launch while on my home network, and it totally worked! Seems like this is an issue with my workplace firewall/antivirus. Thank you for your help!
